.bowerrc

```diff
@@ -1,4 +1,5 @@
 {
+  "registry": "https://registry.bower.io",
```

Comment:
Bower is deprecated and they suggest to migrate away from it: https://bower.io/blog/2017/how-to-migrate-away-from-bower/

We could get this PR verified and merged to not rely on it anymore: https://github.com/aerogear/unifiedpush-admin-ui/pull/29/files

Reply:
@pb82 https://issues.jboss.org/browse/AEROGEAR-8897

Reply:
This meant to be just workaround to make build pass for checks.